src/pages/stake/delegations/Delegations.tsx

```diff
@@ -110,10 +112,17 @@ const Delegations = () => {
             title: t("Validator"),
             dataIndex: "validator_address",
             render: (address: AccAddress, r) => {
-              if (address === allianceHub.useHubAddress()) {
+              if (address === hubAddress) {
                 return <AllianceHubStakeCTA denom={r.balance.denom} />
               } else {
-                return <ValidatorLink address={address} internal img />
+                return (
+                  <StakingAssetLink
```

Comment:
Validator links navigate to delegate page instead of validator details page like in production:

https://github.com/terra-money/station/assets/30275692/111a27d1-9e49-4914-ba57-dcfe2cfd6d35